packages/mui-material/src/Accordion/Accordion.js

```diff
@@ -46,7 +47,7 @@ const AccordionRoot = styled(Paper, {
     ];
   },
 })(
-  ({ theme }) => {
+  memoTheme(({ theme }) => {
     const transition = {
       duration: theme.transitions.duration.shortest,
```

Comment:
There's a problem with prettier, it's going to forcefully increase the indentation level everywhere inside `memoTheme(({ theme }) => /* here */)`, which is one of the reasons I dislike prettier. Anyway, I've left the PR without prettifying it to easy the review, but I'll run it as a last step before merging.